docs/permissions.md

```diff
@@ -0,0 +1,77 @@
+# Permissions in Apply
+
+## Background
+
+Due to the complexity of user and organisational permissions, knowledge sharing between members of the team has not been as straightforward. This document is an attempt to rectify that and enable all developers to gain a thorough understanding of how permissions work across the application.
+
+## Available Permissions
+
+The following set of permissions are used across the app.
+
+### Related to org management
+
+- Manage Users
+
+  Enables a provider user to set invite or delete users and set their permissions.
+
+- Manage Organisations
+
+  Enables a provider user to manage/change permissions between different organisations.
+
+
```

Comment:
I think the extra line is not rendering the heading correctly

Reply:
It's not. I wasn't sure how to resolve that, but also I need to check if that's an issue on redcarpet as well as that's what the tech docs use.

Reply:
hmm your right, putting this through normal markdown linters seems to work, maybe its a Github thing 🤔